DSCResources/cMsmqQueuePermissionEntry/cMsmqQueuePermissionEntry.psm1

```diff
@@ -363,4 +402,38 @@ function Reset-cMsmqQueueSecurity
     }
 }
 
+function Build-QueuePath()
+{
+    <#
+    .SYNOPSIS
+        Build queue path from a MSMQ queue name and type.
+    .DESCRIPTION
+        The Build-QueuePath function builds the queue path from a MSMQ queue name and type.
+    .PARAMETER Name
+        Specifies the name of the queue.
+    .PARAMETER QueueType
+        Specifies the queue type of the queue.
+    #>
+    [CmdletBinding()]
+    param
+    (
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [ValidateNotNullOrEmpty()]
+        [String]
+        $Name,
+
+        [Parameter(Mandatory = $false)]
+        [ValidateSet('Private', 'Public')]
+        [String]
+        $QueueType = 'Private'
+
+    )
+    if ($QueueType -eq 'Private')
+    {
+        $QueuePath = '{0}\{1}$\{2}' -f $env:COMPUTERNAME, $QueueType, $Name
+    }else{

```

Comment:
For consistency with the rest of the code, please place opening and closing braces on their own lines.